src/main/resources/application.yaml

```diff
@@ -27,3 +27,22 @@ spring:
     abandonWhenPercentageFull: 0
     testOnBorrow: true
     validationQuery: SELECT 1
+
+quartzProperties:
+  org.quartz:
+    scheduler:
+      instanceId: AUTO
+    dataSource:
+      jobscheduler:
+        driver: org.postgresql.Driver
+        URL: jdbc:postgresql://${JOB_SCHEDULER_DB_HOST:job-scheduler-database}:${JOB_SCHEDULER_DB_PORT:5432}/jobscheduler
```

Comment:
Maybe there is a way in Spring for one property to reference another? (no idea, generally curious)

Reply:
I'll look into this

Reply:
Done, it works!